server/create_tiff.py

```diff
@@ -0,0 +1,25 @@
+# A romanesco script to convert a slide to a TIFF using vips.
+import subprocess
+import os
+
+out_path = os.path.join(_tempdir, out_filename)
+
+convert_command = (
+    'vips',
+    'tiffsave',
+    '"%s"' % in_path,
+    '"%s"' % out_path,
+    '--compression', 'jpeg',
+    '--Q', '90',
```

Comment:
Also, we should eventually store these parameters, in addition to the fact that it was created with this specific version of this script somewhere in the item (or file?) metadata, to provide provenance.

It's not critical that we get this into the first version of this functionality. We can always add it later.